prymer/primer3/primer3_parameters.py

```diff
@@ -219,6 +219,7 @@ def to_input_tags(self) -> dict[Primer3InputTag, Any]:
             Primer3InputTag.PRIMER_INTERNAL_MAX_GC: self.probe_gcs.max,
             Primer3InputTag.PRIMER_INTERNAL_MAX_POLY_X: self.probe_max_polyX,
             Primer3InputTag.PRIMER_INTERNAL_MAX_NS_ACCEPTED: self.probe_max_Ns,
+            Primer3InputTag.PRIMER_NUM_RETURN: self.number_probes_return
```

Comment:
How hard would it be to implement a unit test to catch a potential regression in the future?

Reply:
I agree this would be nice, but there is not an existing test to amend, and I don't have the time to _also_ fix the test shortcomings at this moment.